apps/ais/.browserslistrc

```diff
@@ -14,3 +14,5 @@ last 2 iOS major versions
 Firefox ESR
 not dead
 not IE 11 # Angular supports IE 11 only as an opt-in. To opt-in, remove the 'not' prefix on this line.
+not ios_saf 15.2-15.3
```

Comment:
Hier gibt es wohl noch immer build-Probleme, oder?
Hat das mit Ivy zu tun? Das war ja schon ein Thema beim Angular Upgrade.

Reply:
Yo das ist ein Bug von Angular. Ich habe alle Möglichkeiten ausprobiert die ich im Internet recherchiert haben konnte aber das hat nicht geklappt. Ich würde vorschlagen, wir machen gleich die Angular-Upgrade. Wenn ich nicht irre, ist dieses Problem schon bei der neuen Version schon behoben.

Reply:
I'm sorry for that 😓 Falls Du am Wochenende am Update arbeiten willst - ich habe relativ viel Zeit und kann vielleicht mithelfen.